crates/re_dev_tools/src/build_search_index/meili.rs

```diff
@@ -194,7 +194,9 @@ impl Task {
             TaskStatus::Succeeded => Ok(ControlFlow::Break(())),
 
             TaskStatus::Failed => {
-                anyhow::bail!("task failed: {}", self.error.as_ref().unwrap().message)
+                #[allow(clippy::unwrap_used)] // unwrap really shouldn't panic here
```

Comment:
nit: this code isn't adding any information

```suggestion
                #[allow(clippy::unwrap_used)]
```